### PR TITLE
setup flake8 lint script

### PR DESCRIPTION
**Pull request by jairhenrique**
@tomchristie 
What kind of check do you want in flake8? 
Do you prefer a file with the validations or parameters in the command line?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,9 @@ install:
     - pip install codecov
     - export PYTHONPATH=.
 
-script: ./scripts/test --cov
+script:
+  - ./scripts/lint --flake8
```
Comment (tomchristie) [inline .travis.yml]: Ideally Travis would run the Linting as a seperate test run. (Eg see how the Travis builds against django rest framework look)

File: .isort.cfg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[settings]
+atomic=true
+multi_line_output=5
+known_standard_library=types
+known_third_party=pytest
+known_first_party=apistar
```
Comment (jairhenrique) [inline .isort.cfg]: based on `django-rest-framework`.

File: .flake8 [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+ignore=E501
```
Comment (jairhenrique) [inline .flake8]: based on `django-rest-framework`

**Comment by tomchristie** [comment]
Ace, thank you!

**Pull request merged**